packages/components/stories/icon/icons.js

```diff
@@ -877,4 +876,4 @@ export const icons =
         'IcWalletZingpayDark',
         'IcWalletZingpayLight',
     ],
-};
```

Comment:
icons file should not be added to your PR since there are no changes to it except that your formatter added a semi  colon to the end of the file.

Reply:
I didn't change this file, and it has been changed by building `deriv-app` and the eslint rules of the project.